crawler/config.py

```diff
@@ -9,6 +9,16 @@ class RSSConfig:
     links = {
         "The Verge": "https://www.theverge.com/rss/index.xml",
         "Engadget": "https://www.engadget.com/rss.xml",
-        # Feedburner based XML Feed, need to be careful
+        "LifeHacker": "https://lifehacker.com/rss",
+        "Wired": "https://www.wired.com/feed/rss",
+        # TODO(@ethkim): Fix the XML feed consumption to prevent breaking from browser-friendly style codes
```

Comment:
Line break after 80 chars. 

And TODO(@xxxx) should contain the person who added the TODO (and who will fix it later), not me. :)

Reply:
I'll fix it.